compiler/luci/logex/src/FormattedGraph.cpp

```diff
@@ -1427,14 +1427,14 @@ bool CircleNodeSummaryBuilderBase::build(const loco::Node *node, locop::NodeSumm
     s.comments().append("[" + cnode->name() + "] = " + ptr_to_str(node));
   };
 
-#define CIRCLE_NODE(OPCODE, CLASS)                     \
-  if (dynamic_cast<const CLASS *>(node))               \
-  {                                                    \
-    if (summary(dynamic_cast<const CLASS *>(node), s)) \
-    {                                                  \
-      add_comment();                                   \
-      return true;                                     \
-    }                                                  \
+#define CIRCLE_NODE(OPCODE, CLASS)                           \
+  if (dynamic_cast<const luci::CLASS *>(node))               \
```

Comment:
Instead of this change, could you check with adding `using namespace luci` somewhere top of this file?